onnxruntime/core/mlas/lib/pooling.cpp

```diff
@@ -1363,10 +1368,20 @@ Return Value:
                 __m128i InputVector0 = _mm_loadu_si128((const __m128i*)&Input[k][ChannelOffset]);
                 __m128i InputVector1 = _mm_loadu_si128((const __m128i*)&Input[k][ChannelOffset + 16]);
 
+                if constexpr (std::is_signed<T8Bits>::value) {
+                    InputVector0 = _mm_xor_si128(InputVector0, BitFlipVector);
+                    InputVector1 = _mm_xor_si128(InputVector1, BitFlipVector);
+                }
+
                 MaximumVector0 = _mm_max_epu8(MaximumVector0, InputVector0);
```

Comment:
maybe signed version should use _mm_max_epi8 and we can get away without bit flipping?